### PR TITLE
LuaForWindows.rjpcomputing version 5.1.5-52

### PR DESCRIPTION
**Pull request by A-Metal-Penguin**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/57558)

### CODE REVIEW

File: manifests/l/LuaForWindows/rjpcomputing/5.1.5-52/LuaForWindows.rjpcomputing.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: LuaForWindows.rjpcomputing
+PackageVersion: 5.1.5-52
+PackageLocale: en-US
+Publisher: rjpcomputing
+PackageName: LuaForWindows
+License: https://github.com/rjpcomputing/luaforwindows/blob/master/LuaForWindows.iss
```
Comment (Trenly) [inline manifests/l/LuaForWindows/rjpcomputing/5.1.5-52/LuaForWindows.rjpcomputing.locale.en-US.yaml:9]: An inno setup file is not a license
Please provide the actual license terms

File: manifests/l/LuaForWindows/rjpcomputing/5.1.5-52/LuaForWindows.rjpcomputing.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: LuaForWindows.rjpcomputing
```
Comment (Trenly) [inline manifests/l/LuaForWindows/rjpcomputing/5.1.5-52/LuaForWindows.rjpcomputing.locale.en-US.yaml:4]: Better to use publisher first
```suggestion
PackageIdentifier: rjpcomputing.LuaForWindows
```

File: manifests/l/LuaForWindows/rjpcomputing/5.1.5-52/LuaForWindows.rjpcomputing.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: LuaForWindows.rjpcomputing
```
Comment (Trenly) [inline manifests/l/LuaForWindows/rjpcomputing/5.1.5-52/LuaForWindows.rjpcomputing.installer.yaml:4]: Better to use publisher first
```suggestion
PackageIdentifier: rjpcomputing.LuaForWindows
```

File: manifests/l/LuaForWindows/rjpcomputing/5.1.5-52/LuaForWindows.rjpcomputing.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: LuaForWindows.rjpcomputing
```
Comment (Trenly) [inline manifests/l/LuaForWindows/rjpcomputing/5.1.5-52/LuaForWindows.rjpcomputing.yaml:4]: Better to use publisher first
```suggestion
PackageIdentifier: rjpcomputing.LuaForWindows
```

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/l/LuaForWindows/rjpcomputing/5.1.5-52/LuaForWindows.rjpcomputing.locale.en-US.yaml (hunk 2 above)
Comment (Trenly) [reply manifests/l/LuaForWindows/rjpcomputing/5.1.5-52/LuaForWindows.rjpcomputing.locale.en-US.yaml:4]: When changing the package ID you will also need to change the names of the file and the folder they are in

**Comment by A-Metal-Penguin** [comment]
I would like to resubmit with changes suggest to me by the community.

**Pull request closed without merging**